### PR TITLE
10191 add module tasks_modification_wizard

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

**Comment by duanyp1991** [comment]
LGTM

**Comment by Rona111** [comment]
cc @elicoidal @seb-elico @duanyp1991 @siyuancn

File: tasks_modification_wizard/README.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========================
+Tasks modification wizard
+=========================
+
+
+This module extends the functionality of project to support ... and allow you to ...
```
Comment (siyuancn) [inline tasks_modification_wizard/README.rst]: support what?
allow user to do what?

Comment (elicoidal) [inline tasks_modification_wizard/README.rst]: "This module extends the functionality of project and adds 
a wizard to mass editing basic information for the tasks".

**Comment by elicoidal** [comment]
Modify the description. Other than that LGTM

**Comment by Rona111** [comment]
cc @elicoidal @siyuancn

**Comment by siyuancn** [comment]
LGTM

File: tasks_modification_wizard/README.rst [hunk 2]
```diff
@@ -0,0 +1,48 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========================
+Tasks modification wizard
+=========================
+
+This module extends the functionality of project and adds a wizard to mass editing basic information for the tasks".
```
Comment (elicoidal) [inline tasks_modification_wizard/README.rst:9]: Remove double quote

**Comment by noahzaozao** [comment]
LGTM

**Pull request merged**